src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditDetails.tsx

```diff
@@ -70,7 +110,7 @@ const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({
             value={userToAutocompleteOption(users.find((user) => user.userId.toString() === projectLead))}
           />
         </Grid>
-        <Grid item xs={12} md={6} sx={{ mt: 1 }}>
+        <Grid item xs={12} md={3} sx={{ mt: 1 }}>
           <FormLabel>Project Manager</FormLabel>
           <NERAutocomplete
```

Comment:
you should change the NERAutocomplete to look like it does in this PR: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/1682 (ignore all the files except for NERAutocomplete)